packages/react-components/src/gallery/dominantSpeaker.ts

```diff
@@ -12,32 +12,42 @@ import { VideoGalleryRemoteParticipant } from '../types';
  * @param visibleParticipants - Array containing currently rendered (visible)
  * participants in the call. {@link @azure/communication-react#VideoGalleryRemoteParticipant}
  * @param maxTiles - Maximum number of tiles to calculate.
+ * @param maxDominantSpeakers - Maximum number of dominant speakers to calculate.
  * @returns VideoGalleryRemoteParticipant[] {@link @azure/communication-react#VideoGalleryRemoteParticipant}
  */
 export const smartDominantSpeakerParticipants = (
   participants: VideoGalleryRemoteParticipant[],
   dominantSpeakers: Array<string> = [],
   visibleParticipants: VideoGalleryRemoteParticipant[] = [],
-  maxTiles = 4 // For video tiles, 4 is the recommended value by Calling team.
+  maxTiles = 4, // For video tiles, 4 is the recommended value by Calling team.,
```

Comment:
There's too many parameters to be able to discern what is being passed in, e.g.
![image](https://user-images.githubusercontent.com/2684369/136137036-78cfc3f9-b8f0-41ba-9b78-366c4bad45b2.png)
Can you update this to be a named object instead of 5 separate params